File: app/src/main/java/com/vano/myrestaurant/model/util/ActivityUtil.java
package com.vano.myrestaurant.model.util;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.vano.myrestaurant.R;

public class ActivityUtil {

    public static void configureActionBar(AppCompatActivity activity, String title) {
        Toolbar toolbar = activity.findViewById(R.id.toolbar);
        activity.setSupportActionBar(toolbar);

        final ActionBar actionBar = activity.getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
            actionBar.setTitle(title);
        }
    }
}

File: app/src/main/java/com/vano/myrestaurant/model/util/ActivityUtil.kt
package com.vano.myrestaurant.model.util

import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.vano.myrestaurant.R

object ActivityUtil {

    fun configureActionBar(activity: AppCompatActivity, title: String?) {
        val toolbar = activity.findViewById<Toolbar>(R.id.toolbar)
        activity.setSupportActionBar(toolbar)
        val actionBar = activity.supportActionBar

        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true)
            actionBar.title = title
        }
    }
}